handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,266 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.
+
+You can create DataPlugins using C++, VBS, LabVIEW or Python.
+
+## Get Started
+
+Python DataPlugins consist of only one Python file that contains all the logic. Almost all language features of the official Python 3.5.9 and its base libraries can be used.
+
+### Plugin class
+
+Start writing your DataPlugin by implementing the
+```python 
+class Plugin:
+```
+The class name cannot be changed.
+
+### Store read
+
+Every Python DataPlugins needs to implement a read_store method. This method is called by DIAdem, LabVIEW or SystemLink DataFinder when it attempts to open your data file. The applications pass a set of useful parameters that can be accessed by the parameter array.

```

Comment:
When referencing text that shows up in code use inline code markup. In this case. 

> Every Python DataPlugins needs to implement a `read_store` method.